### ISSUE TITLE
Strange caracters at the end of sentences before ":"

### ISSUE THREAD

**Issue by ldmpub** [bpellin/keepassdroid#320]
See for instance a screen shot where **\uOOAO** are displayed 4 times:
![new 2_keepassdroid](https://user-images.githubusercontent.com/1452146/45512382-e1285500-b79f-11e8-83cc-e6b64437f29d.jpg)

**Comment by ldmpub** [comment]
Can be closed by merging #321

**Comment by hanscappelle** [comment]
thanks for the fix @ldmpub

**Comment by ldmpub** [comment]
You're welcomed ... merge still pending.

**Comment by hanscappelle** [comment]
Yes it's up to someone with admin rights to review and merge these changes.

**Comment by ldmpub** [comment]
Corrected, see => @bpellin merged commit [883c707](https://github.com/bpellin/keepassdroid/commit/883c7071076f2e7253f890ecc49848c6e4ad705e) into bpellin:master [on Dec 27, 2018](https://github.com/bpellin/keepassdroid/pull/321#event-2045261939).

I close this issue.
